packages/app/src/hooks/useObjectURL.ts

```diff
@@ -0,0 +1,17 @@
+import { useLayoutEffect, useMemo } from 'react';
+
+export function useObjectURL(object: string | ArrayBuffer | Uint8Array | Blob) {
+  const objectURL = useMemo(() => {
+    if (typeof object === 'string') return object;
```

Comment:
We probably don't really want strings here, cause we don't use `createObjectURL`. Or at least we shouldn't call `revokeObjectURL` if the object is a string. If we make the change below about default patterns, we may not need to handle strings here (yet - Sketch does support base64 data, but we can come back to that).